dl/src/main/scala/com/intel/analytics/bigdl/utils/Engine.scala

```diff
@@ -322,14 +349,93 @@ object Engine {
       }
       Some(sc)
     } else {
-      physicalCoreNumber = cores
+      _onSpark = false
```

Comment:
@jason-dai If user don't use bigdl.sh, we output an warning. Change it to an Error?